fission/function.go

```diff
@@ -41,6 +43,55 @@ func fileSize(filePath string) int64 {
 	return info.Size()
 }
 
+func getFnPkgRef(client *client.Client, fnName string, fileName string) fission.FunctionPackageRef {
```

Comment:
1. Please add a comment saying what this function does.
2. I think it should be renamed to something like "createPackageFromFile" to more accurately reflect what it's doing